api/src/functionalTest/java/FunctionalTest.java

```diff
@@ -25,7 +25,7 @@ public boolean isOneTimePasswordValid(String base32Key, String token) {
     };
 
     private AuthService authService = new AuthService(
-        new JwtTool(
+        new JwtHS512Tool(
```

Comment:
Not sure why it works  with `JwtTool` on sandbox env but not on saat?

Reply:
My guess is that `functional` task wasn't executed in that pipeline

Reply:
I've checked and that seems to be the case